vision/unit_tests/module_in_frame/UnitTest.py

```diff
@@ -0,0 +1,26 @@
+"""
+This file is used for testing ModuleInFrame
+"""
+
+import unittest
+from ModuleInFrame import ModuleInFrame as mif
+
+class TestModuleInFrame(unittest.TestCase):
+    def test_ModuleInFrame(self):
+        """
+        Testing ModuleInFrame
+
+        Returns
+        -------
+        ndarray[bool] which images the module was detected in
+        """
+
+        results = []
+        expected_results = [True, True, True, True]
```

Comment:
Lets add some extra non-module images to see if the algorithm gives false positives.